CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+  - [573](https://github.com/thoth-pub/thoth/pull/573) - Add seconds to timestamp for Crossref metadata output
```

Comment:
Where we have an issue number, we usually put that rather than the PR number (we've been guilty of a lot of PRs lately with no issues attached!)